Frontend-path.md

```diff
@@ -89,12 +89,53 @@ Here you can take any "big" course so you understand the basics of JS. We have l
 
 **🤝 What we exepct from you to know and understand**
 - Syntax
+- Logging and Debugging
 - Data types
+- Variable declaration and initialisation with var, let, const
+- Scope
+- Comparisons
+- Type conversion, particularly to boolean
+- Logical Operators
+- Conditions
+- Functions
+  - return values of functions
+  - null vs. undefined
+  - conditional returns
+  - parameters & their scope
+  - default params
+  - destructuring parameters
+  - rest operator
+  - callback functions
 - Objects
+  - creating and manipulating objects (adding properties, deleting properties, extracting properties)
+  - nesting objects and objects with functions
+  - understanding referential vs. primitive types
+  - copying objects
+  - spread operator
+  - destructuring objects and setting default values
+  - iterating over object properties/values
+  - conditional properties
+  - computed properties
 - Arrays
-- Functions
-- Asyncronous JS - Promises, Async/Await
+  - creating and manipulating arrays (adding elements, removing elements, finding elements, splitting arrays, joining arrays)
+  - copying arrays
+- map, filter, reduce
+  - learn when to use map, filter and reduce
+  - functional vs. procedural programming
+- Closure
+  - understanding what problems the concept solves
+- Asyncronous JS
+  - creating promises
+  - then and callbacks vs. async/await syntax
+  - understanding the event loop
 - Error handling
+- ES6 imports & dynamic imports
```

Comment:
Here it would be nice to cover not only imports, but overall modular approach to our files, code splitting, lazy loading (that's the dynamic imports I guess)

Reply:
Also the difference with `require` so at least they know what's happening when they're using external libs